arangod/Graph/PathType.cpp

```diff
@@ -33,6 +33,7 @@ namespace graph {
 constexpr char const* KShortestPathsName = "K_SHORTEST_PATHS";
 constexpr char const* KPathsName = "K_PATHS";
 constexpr char const* AllShortestPathsName = "ALL_SHORTEST_PATHS";
+constexpr char const* ShortestPathName = "SHORTEST_PATH";
```

Comment:
nit: the constant naming here is inconsistent.
According to our coding conventions, all names should  `k...Name`.
Apart from that, these `char const*`s should be `std::string_views`.
That doesn't need to be fixed in 3.10, but can be addressed in devel only.